example/index.html

```diff
@@ -279,6 +279,10 @@
 					url: 'https://raw.githubusercontent.com/gkjohnson/3d-demo-data/main/models/botanists-greenhouse/scene.gltf',
 					credit: 'Model by "riikkakilpelainen" on Sketchfab.',
 				},
+				'Vertex Colors Test': {
+					url: 'https://raw.githubusercontent.com/alvinalexander/3d-demo-data/main/models/vertex-color-cube.glb',
+					credit: 'Model by aghouas on Sketchfab.',
+				},
```

Comment:
I think instead of adding a new test model to the main demo we can use the model-viewer page for testing. See these two model pages here which can be run locally:

- https://gkjohnson.github.io/three-gpu-pathtracer/example/bundle/viewerTest.html#khronos-BoxVertexColors
- https://gkjohnson.github.io/three-gpu-pathtracer/example/bundle/viewerTest.html#khronos-VertexColorTest